engine/src/operators/OrderBy.cpp

```diff
@@ -321,9 +321,187 @@ void process_sort(blazing_frame & input, std::string query_part, Context * query
 	}
 }
 
+
+
+}  // namespace operators
+}  // namespace ral
+
+namespace ral {
+namespace operators {
+namespace experimental {
+
+	using Context = blazingdb::manager::experimental::Context;
```

Comment:
remove this and change to a typedef as we agreed

Reply:
we should apply typedefs and all that after the migration is complete. Otherwise we are doing more temporary work we dont need to

Reply:
Its not more work its actually equivalent work

Reply:
Could you explain what typedef you want and where it would go. I can try it out and see if what you are asking will work. My main concern is that a typedef will not work while there are more than one object called Context. Because there is 
`blazingdb::manager::experimental::Context` and 
`blazingdb::manager::Context`

Reply:
so I am doing this for example:

```typedef cudf::experimental::table CudfTable;```

another example i guess is what they do in cudf which is

```namespace cudf_io = cudf::experimental::io;```

I am not picky about which one we use but here are 3 ways to do the same thing and we should use 1 across the board.

Reply:
ok, let me see what I can do

Reply:
Another drive by comment, `using` is the preferred modern C++ syntax for type aliases as opposed to `typedef` these days.